docs/src/pages/guides/aliases.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Aliases
+---
+
+Aliasing is an integral part when it comes to clean and readable code, that's why Astro comes with support for custom aliases out of the box.
+
+## What are aliases?
+
+Aliases are _keywords_ you can use instead of paths. 
+So instead of `../../components/SomeComponent/` you can use `components/SomeComponent`. Cool, right!
```

Comment:
Sorry dude, this is a form of narrative out with the formality used in the docs, gutted I'm the one to do that, 😢
```suggestion
So instead of `../../components/SomeComponent/` you can use `components/SomeComponent`. 
```

Reply:
Fair enough. 🤷